docs/index.rst

```diff
@@ -36,20 +65,107 @@ The plugin may be used within a north *task* or *service*. Both of these are cre
 
             {
-                "sensor": {
-                    "temperature": {
-                        "node": "ns=1;i=1013",
+                "sinusoid": {
+                    "sinusoid": {
+                        "node": "ns=3;i=1017",
                         "type": "Float"
                     }
                 }
             }
 
-        - sensor is an asset name
-        - temperature is a datapoint name
+        - sinusoid is an asset name
+        - sinusoid is a datapoint name
 
       - **Source**: The source of the data to be sent, this may be the *readings* or *statistics* data.
 
-  - Click *Next*
+The OPC UA Security tab contains a set of configuration items that is used for setting the security between the client and the OPC UA Server.
+
++------------------------+
+| |opcuaclient_security| |
++------------------------+
+
+  - **Security Mode**: Specify the OPC/UA security mode that will be used to communicate with the OPC/UA server.
+
+    +-----------------+
+    | |security_mode| |
+    +-----------------+
+
+  - **Security Policy**: Specify the OPC/UA security policy that will be used to communicate with the OPC/UA server.
+
+    +-------------------+
+    | |security_policy| |
+    +-------------------+
+
+  - **User Authentication Mode**: Specify the user authentication mode that will be used when authenticating the connection to the OPC/UA server.
+
+    +----------------------------+
+    | |user_authentication_mode| |
+    +----------------------------+
+
+  - **Username**: Specify the username to use for authentication. This is only used if the *User Authentication Mode* is set to *Username And Password*.
+
+  - **Password**: Specify the password to use for authentication. This is only used if the *User Authentication Mode* is set to *Username And Password*.
+
+  - **Server Public Certificate**: (Optional) The name of the public certificate of the OPC/UA server specified in the *OPCUA Server URL*. This must either be a DER or PEM format certificate file.
+
+  - **Client Public Certificate**: The name of the public certificate of the OPC/UA client application, that is, this plugin. This must either be a DER or PEM format certificate file.
+
+  - **Client Private Key**: The name of the private key of the client application, that is, the private key the plugin will use. This must be a PEM format key file.
+
+  - **Client Private Passphrase Key**: (Optional) The passphrase of the private key of the client application.
+
+
+- Click *Next*
+
+- Enable your task or service
+
+- Click *Done*
+
+
+Certificate Management
+----------------------
+
+Typically the Certificate Authorities certificate is retrieved and uploaded to the Fledge Certificate Store along with the certificate from the OPC/UA server that has been signed by that Certificate Authority. A public/private key pair must also be created for the plugin and signed by the Certificate Authority. These are uploaded to the Fledge Certificate Store.
```

Comment:
Explain how self-signed certificates are different. They are not signed by a Certificate Authority. The system manager must copy the server certificate to the Fledge Certificate Store, and the client certificate to the server's trusted certificate store.